test/RunTests

```diff
@@ -1710,4 +1710,11 @@ printf '3 |f a b c |e x y z\n2 |f a y c |e x\n' | {VW} --oaa 3 -q ef --audit
 {VW} -k --oaa 10 --oaa_subsample 5 -c --passes 10 -d train-sets/multiclass --holdout_off
     train-sets/ref/oaa_subsample.stderr
 
+# test 191: online contextual memory tree 
+{VW} train-sets/rcv1_small.dat --memory_tree 10 --learn_at_leaf 1 --max_number_of_labels 2 --dream_at_update 0 --dream_repeats 3 --oas 0 --online 1 --leaf_example_multiplier 10 --alpha 0.1 -l 0.001 -b 29 --passes 1 --loss_function squared --holdout_off train-sets/ref/cmt_rcv1_small_online.stderr
```

Comment:
In order for Windows test generation to pick up this test, the stderr reference file needs to be put on the next line.

```suggestion
{VW} train-sets/rcv1_small.dat --memory_tree 10 --learn_at_leaf 1 --max_number_of_labels 2 --dream_at_update 0 --dream_repeats 3 --oas 0 --online 1 --leaf_example_multiplier 10 --alpha 0.1 -l 0.001 -b 29 --passes 1 --loss_function squared --holdout_off
    train-sets/ref/cmt_rcv1_small_online.stderr
```